### PR TITLE
refactor claims database context

### PR DESCRIPTION
**Pull request by Luke-Clements**
# Description

Refactoring claims so that DatabaseContext is passed through layers and all database operations are in the same transaction.

## Type of change

chore

# How Has This Been Tested?

NOTE: Units tests had to be updated to work with this change.
All Units tests passed. 

# Notes

.25 hr

### CODE REVIEW

File: Backend/UnitTesting/ClaimServiceUT.cs [hunk 1]
```diff
@@ -1,96 +1,90 @@
...
-            bool hasAccess = claimService.UserHasServiceAccess(user1, service2);
-
-            Assert.IsFalse(hasAccess);
-        }
-    }
-}
+using DataAccessLayer.Database;
+using DataAccessLayer.Models;
+using System;
+using System.Linq;
```
Comment (alfredovargasjr) [inline Backend/UnitTesting/ClaimServiceUT.cs:4]: Whats difference from this UT and the other Claims UT from the other PR?

**Review by alfredovargasjr — CHANGES_REQUESTED** [review]

File: Backend/UnitTesting/ClaimServiceUT.cs (hunk 1 above)
Comment (Luke-Clements) [reply Backend/UnitTesting/ClaimServiceUT.cs:4]: Refactored so DatabaseContext is passed into ManagerLayer functions and also removed extra lines that were not needed.

File: Backend/UnitTesting/app.config [hunk 2]
```diff
@@ -1,40 +1,25 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<configuration>
-  <configSections>
```
Comment (ChrisMeyer7088) [inline Backend/UnitTesting/app.config:3]: should not have changed the app.config dependencies

**Review by ChrisMeyer7088 — CHANGES_REQUESTED** [review]

File: Backend/UnitTesting/app.config (hunk 2 above)
Comment (alfredovargasjr) [reply Backend/UnitTesting/app.config:3]: This app.config looks like the ones in the controller project lol

Comment (alfredovargasjr) [reply Backend/UnitTesting/app.config:3]: We just have to make sure the public key for the DAL is in there

**Pull request closed without merging**